dev-docs/bidders/equativ.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: bidder
+title: Equativ
+description: Prebid Equativ Bidder Adapter
+biddercode: equativ
+media_types: banner
+tcfeu_supported: true
+coppa_supported: true
+gvl_id: 45
+gpp_supported: true
+schain_supported: true
+usp_supported: true
+userIds: all
+pbjs: true
+pbs: true
+floors_supported: true
+sidebarType: 1
+---
+
+### Registration
+
+The Equativ bidder adapter requires setup and approval from the Equativ service team. Please reach out to your account manager for more information to start using it.
+
+### Bid params
+
+{: .table .table-bordered .table-striped }
+
```

Comment:
please remove the extra space here

Reply:
By extra space I'm assuming you mean the line between the table definition and the content.  Marking as resolved.